main.js

```diff
@@ -105,22 +114,43 @@
     playerOneDisplay.innerText = challengerNames[0].value;
     playerTwoDisplay.innerText = challengerNames[1].value;
     disableInput()
+    guessErrorMessage()
     checkGuesses()
-    guessOneDisplay.innerText = challengerGuesses[0].value;
-    guessTwoDisplay.innerText = challengerGuesses[1].value;
     clearFields()
     disableButtons()
     gameResetButton.removeAttribute('disabled')
   }
 
+  function guessErrorMessage() {
+    console.log(maxNumber)
+    if (parseInt(challengerGuesses[0].value) > maxNumber ||
+        parseInt(challengerGuesses[0].value) < minNumber) {
+          guessOneDisplay.innerText = 'N/A';
+          if (numberOfGuesses !== 0) {
+              numberOfGuesses -= 1;
+          };
+    } else {
+      guessOneDisplay.innerText = challengerGuesses[0].value;
+    } if (parseInt(challengerGuesses[1].value) > maxNumber ||
+        parseInt(challengerGuesses[1].value) < minNumber){
+          guessTwoDisplay.innerText = 'N/A';
+          if (numberOfGuesses !== 0) {
+              numberOfGuesses -= 1;
```

Comment:
This becomes unnecessary if we disable submission

Reply:
cut it!